streaming/index.js

```diff
@@ -661,7 +758,12 @@ const startServer = async () => {
         eventHandlers.onKill();
       } else if (event === 'filters_changed') {
         req.log.debug(`Invalidating filters cache for ${req.accountId}`);
-        req.cachedFilters = null;
+
+        fetchFilters(req.accountId).then((filters) => {
+          req.cachedFilters = filters;
+        }).catch((err) => {
+          req.log.error({ err }, 'Error refreshing filters');
```

Comment:
I think in this likely rare scenario, we'd want to leave the old filters in place, rather than wipe out all filters entirely.